content/verification/address-validation/release-notes/2024-05-15.md

```diff
@@ -0,0 +1,10 @@
+Released version 3 of address validation. Updates include:
+- `/validate` and `/candidate` endpoints
+
+     - Removed support for US Congressional District. 
+     - Added country code support for both International Standards Organization (ISO)  and Federal Information Processing (FIPS) standards.
+
+- `/cityStateProvince` endpoint is renamed to `/locality` to reflect the capability.
```

Comment:
Reflect the capability of what?

Reply:
reflects the capability of the endpoint.